pkg/restapi/v1/verifier/openapi.gen.go

```diff
@@ -9,42 +9,53 @@ import (
 
 	"github.com/deepmap/oapi-codegen/pkg/runtime"
 	"github.com/labstack/echo/v4"
-)
-
-// Defines values for VerifierChecksCredentialFormat.
-const (
-	JwtVc VerifierChecksCredentialFormat = "jwt_vc"
-	LdpVc VerifierChecksCredentialFormat = "ldp_vc"
-)
-
-// Defines values for VerifierChecksPresentationFormat.
-const (
-	JwtVp VerifierChecksPresentationFormat = "jwt_vp"
-	LdpVp VerifierChecksPresentationFormat = "ldp_vp"
+	externalRef0 "github.com/trustbloc/vcs/pkg/restapi/v1/common"
 )
 
 // Model for creating verifier profile.
 type CreateVerifierProfileData struct {
-	// Type of checks to be performed and formats supported.
-	Checks map[string]interface{} `json:"checks"`
+	// Checks to be performed by a verifier profile for verifying credentials and presentations.
+	Checks VerifierChecks `json:"checks"`
+
+	// Model for KMS configuration.
+	KmsConfig *externalRef0.KMSConfig `json:"kmsConfig,omitempty"`
 
 	// Verifier’s display name.
 	Name string `json:"name"`
 
-	// Configuration for participating in OIDC4VC credential interaction operations.
-	OidcConfig *map[string]interface{} `json:"oidcConfig,omitempty"`
+	// Model for OIDC4VPConfig configuration.
+	OidcConfig *OIDC4VPConfig `json:"oidcConfig,omitempty"`
 
 	// Unique identifier of the organization.
-	OrganizationID string `json:"organizationID"`
+	OrganizationID          string                    `json:"organizationID"`
+	PresentationDefinitions *[]PresentationDefinition `json:"presentationDefinitions,omitempty"`
 
 	// URI of the verifier.
 	Url *string `json:"url,omitempty"`
 }
 
+// Model for OIDC4VPConfig configuration.
+type OIDC4VPConfig struct {
+	// DID method of the DID to be used for signing.
+	DidMethod externalRef0.DIDMethod `json:"didMethod"`
+
+	// Cryptographic algorithms used to sign oipd vp request object.
```

Comment:
oipd -> oidc?

do we need `Ro` prefix in `RoSigningAlgorithm`? if this is important to emphasize "request object" maybe name it something like "RequestSigningAlgorithm" ("ro" has a collocation with "read-only")

Reply:
"Request" is confusing, but yes "ro" is definitely not the best name. 

I will include fix for oipd and "ro" in next PR